molecule/default/verify.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Check whether packages are installed
+  hosts: all
+  become: yes
+
+  tasks:
+    - name: Gather package facts
+      package_facts:
+        manager: auto
+
+    - name: Verify Packages
```

Comment:
Rather than verify package presence, which is already done and verified by `converge` and `idemptotence` steps, let's call these utilties with their "version" flag to see if they're ready for use. If they're installed, working (and called correctly), the task will pass, and if not, it will fail, which is what we want, and is more meaningful than just checking if packages are present.

This can/should be OS agnostic, and we can skip testing any utilities that don't appear on both OS'es:
- mcrypt is a library, no way to call it
- nmon is ubuntu only, and is pretty old, so can be skipped

Not all have version, but do have --help , so test with that instead: 
- nload
- unzip

So something like 
```yaml
- name: Verify utilities are available
  command: >
    {{ item }}
  with_items: 
    - git --version
    - curl --version 
    - htop --version 
    - nload --help
    etc etc etc etc    
```

Reply:
I have modified the verify.yml.  This fails for screen and 'multitail'.

```
failed: [instance] (item=multitail -V) => {"ansible_loop_var": "item", "changed": true, "cmd": ["multitail", "-V", "with_items:"], "delta": "0:00:00.012284", "end": "2020-12-01 11:56:18.126500", "item": "multitail -V", "msg": "non-zero return code", "rc": 1, "start": "2020-12-01 11:56:18.114216", "stderr": "Error opening terminal: unknown.", "stderr_lines": ["Error opening terminal: unknown."], "stdout": "", "stdout_lines": []}
```
```
failed: [instance] (item=screen --version) => {"ansible_loop_var": "item", "changed": true, "cmd": ["screen", "--version", "with_items:"], "delta": "0:00:00.423396", "end": "2020-12-01 12:21:04.672931", "item": "screen --version", "msg": "non-zero return code", "rc": 1, "start": "2020-12-01 12:21:04.249535", "stderr": "", "stderr_lines": [], "stdout": "Screen version 4.06.02 (GNU) 23-Oct-17", "stdout_lines": ["Screen version 4.06.02 (GNU) 23-Oct-17"]}
```